src/components/welcome/MyAccounts/index.tsx

```diff
@@ -59,60 +76,89 @@ const AccountsList = ({ safes, onLinkClick }: AccountsListProps) => {
           <Typography variant="h1" fontWeight={700} className={css.title}>
-            Safe accounts
+            My accounts
           </Typography>
-          <Track {...OVERVIEW_EVENTS.CREATE_NEW_SAFE} label={trackingLabel}>
-            <CreateButton isPrimary={!!wallet} />
-          </Track>
-        </Box>
-
-        <PaginatedSafeList
-          title="My accounts"
-          safes={ownedSafes}
-          onLinkClick={onLinkClick}
-          noSafesMessage={
-            wallet ? (
-              NO_SAFES_MESSAGE
-            ) : (
-              <>
-                <Box mb={2}>Connect a wallet to view your Safe Accounts or to create a new one</Box>
-                <Track {...OVERVIEW_EVENTS.OPEN_ONBOARD} label={trackingLabel}>
-                  <ConnectWalletButton text="Connect a wallet" contained />
-                </Track>
-              </>
-            )
-          }
-        />
-
-        <PaginatedSafeList
-          title={
-            <>
-              <VisibilityOutlined sx={{ verticalAlign: 'middle', marginRight: '10px' }} fontSize="small" />
-              Watchlist
-            </>
-          }
-          safes={watchlistSafes || []}
-          action={
+          <Box className={css.headerButtons}>
             <Track {...OVERVIEW_EVENTS.ADD_TO_WATCHLIST} label={trackingLabel}>
               <Link href={AppRoutes.newSafe.load}>
                 <Button
                   disableElevation
+                  variant="outlined"
                   size="small"
                   onClick={onLinkClick}
                   startIcon={<SvgIcon component={AddIcon} inheritViewBox fontSize="small" />}
+                  sx={{ height: '36px', width: '100%', px: 2 }}
                 >
                   Add
                 </Button>
               </Link>
             </Track>
-          }
-          noSafesMessage={NO_WATCHED_MESSAGE}
-          onLinkClick={onLinkClick}
-        />
+            <Track {...OVERVIEW_EVENTS.CREATE_NEW_SAFE} label={trackingLabel}>
+              <CreateButton isPrimary={!!wallet} />
+            </Track>
+          </Box>
+        </Box>
+
+        <Paper className={css.safeList}>
+          {/* Pinned Accounts */}
+          <Box mb={2} minHeight="170px">
+            <div className={css.listHeader}>
+              <SvgIcon
+                component={BookmarkIcon}
+                inheritViewBox
+                fontSize="small"
+                sx={{ mt: '2px', mr: 1, strokeWidth: 2 }}
+              />
+              <Typography variant="h5" fontWeight={700} mb={2}>
+                Pinned
+              </Typography>
+            </div>
+            {pinnedSafes.length > 0 ? (
+              <SafesList safes={pinnedSafes} onLinkClick={onLinkClick} />
+            ) : (
+              <Box className={css.noPinnedSafesMessage}>
+                <Typography color="text.secondary" maxWidth="350px" textAlign="center" fontSize="14px">
```

Comment:
nit: You can also use `variant="body2"` to achieve 14px font-size instead of setting it separately.